sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java

```diff
@@ -130,10 +130,20 @@
         LocationOptionsQueries locationOptionsQueries = new LocationOptionsQueries();
         locationOptionsQueries.setLocationOptions(locationOptions);
 
+        if (locationOptions == null) {
```

Comment:
Why do you need this change? As long as this is inside the try-catch, we will still return a mono.error, right?

Reply:
Done you are right inside the try will return the mono.error